flytepropeller/pkg/compiler/validators/typing.go

```diff
@@ -19,6 +19,8 @@ type trivialChecker struct {
 }
 
 func removeTitleFieldFromProperties(schema map[string]*structpb.Value) {
+	// TODO: Explain why we need this
+	// TODO: givse me example about dataclass vs. Pydantic BaseModel
```

Comment:
To add the comment, writing the entire schema would make it too lengthy. Would it be acceptable to use something like this instead?

```
class A:
	a: int

Pydantic.BaseModel: 	{"properties": {"a": {"title": "A", "type": "integer"}}}
dataclass: 			{"properties": {"a": {"type": "integer"}}, "additionalProperties": false}
```

Reply:
Are you proposing to preprocess the schemas so that one can mix and match dataclasses and base models given their schemas are aligned? I personally feel this is not necessary and think it would be totally acceptable to consider a dataclass and a base model *not* a match by default. Especially if this makes things a lot more complicated in the backend otherwise. What do you think about this?

Reply:
I think it actually make things more complicated, will remove related logic.